### PR TITLE
Update gluster provisioner parameters

### PR DESCRIPTION
**Pull request by wongma7**
make it match https://github.com/kubernetes/kubernetes/blob/release-1.4/examples/experimental/persistent-volume-provisioning/README.md

@jsafrane

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1226)

### CODE REVIEW

**Comment by devin-donnelly** [comment]
Is this not relevant until 1.4 is released? I'm noticing you're referencing something in release-1.4.

**Comment by wongma7** [comment]
@devin-donnelly yes, so this should be targeting the release-1.4 branch right? It looks like a lot of stuff I did previously that was intended only for 1.4 is in master, so I will fix that in a separate PR...

**Comment by devin-donnelly** [comment]
Yes, correct. This should target release-1.4.

File: docs/user-guide/persistent-volumes/index.md [hunk 1]
```diff
@@ -365,22 +365,24 @@ parameters:
...
+  restauthenabled: "true"
   restuser: "admin"
   restuserkey: "password"
 ```
 
 * `endpoint`: `glusterfs-cluster` is the endpoint/service name which includes GlusterFS trusted pool IP addresses and this parameter is mandatory.
-* `resturl` : Gluster REST service url which provision gluster volumes on demand. The format should be a valid URL and this is a mandatory parameter for GlusterFS dynamic provisioner.
-* `restuser` : Gluster REST service user who has access to create volumes in the Gluster Trusted Pool. This parameter is optional, empty string will be used when omitted.
-* `restuserkey` : Gluster REST service user's password which will be used for authentication to the REST server. This parameter is optional, empty string will be used when omitted.
+* `resturl` : Gluster REST service url which provision gluster volumes on demand. The format should be `IPaddress:Port` and this is a mandatory parameter for GlusterFS dynamic provisioner.
```
Comment (devin-donnelly) [inline docs/user-guide/persistent-volumes/index.md]: provision -> provisions

"this parameter is mandatory when using the GlusterFS dynamic provisioner."

File: docs/user-guide/persistent-volumes/index.md [hunk 2]
```diff
@@ -365,22 +365,24 @@ parameters:
...
+* `restauthenabled` : Gluster REST service authentication boolean is required if the authentication is enabled on the REST server. If this value is 'true', 'restuser' and 'restuserkey' have to be filled.
```
Comment (devin-donnelly) [inline docs/user-guide/persistent-volumes/index.md]: Needs a clarity edit:

"A boolean value that indicates whether Gluster REST service authentication is enabled on the REST server. If this value is 'true', you must supply values for the 'restuser' and 'restuserkey' parameters."

File: docs/user-guide/persistent-volumes/index.md [hunk 3]
```diff
@@ -365,22 +365,24 @@ parameters:
...
+* `restuser` : Gluster REST service user who has access to create volumes in the Gluster Trusted Pool.
```
Comment (devin-donnelly) [inline docs/user-guide/persistent-volumes/index.md]: Comma after "user".

File: docs/user-guide/persistent-volumes/index.md [hunk 4]
```diff
@@ -365,22 +365,24 @@ parameters:
...
+* `restuserkey` : Gluster REST service user's password which will be used for authentication to the REST server.
```
Comment (devin-donnelly) [inline docs/user-guide/persistent-volumes/index.md]: delete "which", comma after "password".

**Comment by wongma7** [comment]
I changed target branch and addressed the feedback in the same commit, looks like it broke github review though. Let me know if I should just open a new PR..

**Comment by devin-donnelly** [comment]
No need, that looks fine. Docs LGTM.

**Comment by devin-donnelly** [comment]
@mikedanese , can you or someone else give this a tech review? It should be short.

**Comment by jsafrane** [comment]
I can review it. There is one minor issue with address format, I'd rather have it precise.

**Comment by devin-donnelly** [comment]
@wongma7 , can you ping me when you've addressed @jsafrane 's feedback? Thanks.

**Comment by wongma7** [comment]
@devin-donnelly addressed feedback, made format more precise

**Comment by jsafrane** [comment]
lgtm from tech side

**Pull request merged**